src/composable/useValidate.js

```diff
@@ -0,0 +1,28 @@
+import { computed } from '@vue/composition-api';
+
+export const validateFields = ({ fields, errors }) => {
```

Comment:
maybe use utils package? because we have this functions in vault as well
f.e. `import { useValidation } from 'utils/compose'`